### ISSUE TITLE
Path of config files in configmap does not matach the real path

### ISSUE THREAD

**Issue by HongofQSH** [StarRocks/starrocks-kubernetes-operator#72]
In the operator source file pkg/fe_controller/fe_pod.go, the variable fe_config_path is defined as "/etc/starrocks/fe/conf". However, when I downloaded the image starrocks/fe-ubuntu:2.5.2, its configuration path was located at /opt/starrocks/fe/conf, which does not match the value of fe_config_path.
I am curious to know why fe_config_path is not defined as "/opt/starrocks/fe/conf".

**Comment by shileifu** [comment]
oh, Thanks for your carefulness, the path is different for compatible docker direct start on VM . the `conf` file build into image when using configmap to map will conflict.

**Comment by HongofQSH** [comment]
I think that the fe.conf shall be fixed in /fe/conf directory under starrocks path, and cannot be set to other paths. Is that right?
When I change the value of the configmap, it should be reflected in the starrocks/fe/conf/fe.conf of the pod, otherwise, it will not have any effect on the fe settings. Therefore, I think the fe_config_path should be set to the actual path of the fe configuration file.

**Comment by shileifu** [comment]
yes, the updated configmap will be effect after the pods restart.(the configmap effect logical) the `entrypoint.sh` will check  configmap exist, if configmap exist it will replace the original conf file from image.

**Comment by kevincai** [comment]
> I think that the fe.conf shall be fixed in /fe/conf directory under starrocks path, and cannot be set to other paths. Is that right? When I change the value of the configmap, it should be reflected in the starrocks/fe/conf/fe.conf of the pod, otherwise, it will not have any effect on the fe settings. Therefore, I think the fe_config_path should be set to the actual path of the fe configuration file.

It is a workaround, we can directly mount fe.conf into /opt/starrocks/fe/conf/, because other than `fe.conf`, there are other conf files there, configmap mounting as a directory, will hide all the contents in the mountpoint, including other conf files.

It would be an issue if the image was running as non-root, and has no access to /etc/starrocks/ directory, we will fix that when we formally support the non-root user mode.
